lib/searchvideoscreative.js

```diff
@@ -49,6 +50,11 @@ class SearchVideosCreative extends GettyApiRequest {
         return webHelper.get(path);
```

Comment:
On the topic of tests, I _think_ you'd see a failure given this line.  I think it needs to be:

``` js
return webHelper.get(path,this.headers);
```

So that the headers get included.  But I don't claim expertise here on JS.  A test will prove this out for you.